heron/packing/src/java/com/twitter/heron/packing/PackingUtils.java

```diff
@@ -136,4 +138,26 @@ public static boolean isValidInstance(Resource instanceResources,
 
     return containerPlans;
   }
+
+  /**
+   * Sort the container plans based on the container Ids
+   * @param containers
+   * @return sorted array of container plans
+   */
+  public static PackingPlan.ContainerPlan[] sortOnContainerId(
```

Comment:
Could you add back the `PackingPlanTest.testContainerSorting` from the earlier review as `PackingUtilsTest.testContainerSorting`?